index.js

```diff
@@ -141,7 +133,7 @@ function validateTimestamp(requestBody) {
 
 
 // certificate validator express middleware for amazon echo
-var verifier = module.exports = function(cert_url, signature, requestBody, callback) {
+var verifier = module.exports = function(cert_url, signature, requestBody, callback, cache) {
```

Comment:
I made it so that if the `cache` arg is omitted, things still work and `globalCache` (see lines 9 & 42) is used as fall-back.